source/js/foundation/inject-react/multipage-nav.js

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import MultipageNavMobile from "../../components/multipage-nav-mobile/multipage-nav-mobile.jsx";
+
+/**
+ * Inject multipage navs (secondary navs)
+ * @param {Array} apps The existing array we are using to to track all ReactDOM.render calls
+ */
+export default apps => {
+  if (document.querySelector(`#multipage-nav`)) {
+    let links = [];
+
+    links = [].map.call(
```

Comment:
omg we need to rewrite this later so badly =D

Reply:
yea I will make sure this gets upated! lol